package.json

```diff
@@ -121,8 +121,8 @@
     "@babel/register": "^7.23.7",
     "@percy/cli": "^1.28.2",
     "@percy/storybook": "^5.0.1",
-    "@skyscanner/bpk-foundations-web": "^17.9.0",
-    "@skyscanner/bpk-svgs": "^19.2.0",
+    "@skyscanner/bpk-foundations-web": "^17.11.0",
+    "@skyscanner/bpk-svgs": "^19.3.0",
```

Comment:
We also need to bump these versions inside the `packages/` folder so consumers will also be able to get these changes without issue too